lib/draper/decorator.rb

```diff
@@ -222,9 +222,9 @@ def attributes
     # @return [Class] the class created by {decorate_collection}.
     def self.collection_decorator_class
       name = collection_decorator_name
-      name.constantize
-    rescue NameError
-      Draper::CollectionDecorator
+      name_constant = name && name.safe_constantize
+
+      name_constant || Draper::CollectionDecorator
```

Comment:
@masolin How do you feel about using the safe navigation operator here? I believe it reads better and is more succint. This could become:

```ruby
name&.safe_constantize || Draper::CollectionDecorator
```

Reply:
If we still want to support Ruby 2.2 and 2.3, then we shouldn't use safe navigation operator, right?

Reply:
Haha, yes good catch. Sorry about that.